index.js

```diff
@@ -22,7 +22,10 @@ module.exports = {
                 end: "$$"
             },
             process: function(blk) {
-                return katex.renderToString(blk.body);
+                var tex = blk.body;
+                var display = tex[0] === "\n";
```

Comment:
Borrowed (in inverted form) from [`plugin-mathjax`](https://github.com/GitbookIO/plugin-mathjax/blob/9745b4154579d8001e59f024bde791a9996ca3c4/index.js#L59), detects the following as display mode:

```
$$
i = \sqrt{-1}
$$
```